### ISSUE TITLE
prototype

### ISSUE THREAD

**Issue by shresdee** [chaudhary6-1/GameZone#2]
Dear client,
I have created a prototype for the website; therefore, please provide me further info to update the website. Additionally, I highly anticipate your preferred font, color and overall look. 
Thank you,
Prototype: https://www.figma.com/file/e0rgHbhAgByXzOhLsx7DYZ/Gamezone?node-id=0%3A1

**Comment by obadaghozlan** [comment]
Hello shresdee,
Thanks for the prototype, please consider moving the About us, Contact us up in the navigation bar as shown in the prototype we sent in the ReadMe file.
Also please consider to add a background picture to new releases, most awaited games and top rated games.
best regards
Obada Ghazlan

**Comment by shresdee** [comment]
Dear client,
I have posted an updated version; so let me know about the changes you desire.
prototype: https://www.figma.com/file/e0rgHbhAgByXzOhLsx7DYZ/Gamezone?node-id=0%3A1
Thank you and warm regards.

**Comment by chaudhary6-1** [comment]
Hello shresdee,
I dont think there is need for login/signup window being shown all the time. Other than that it looks ok to me. 
regards,
Zain Khurram Chadhary

**Comment by shresdee** [comment]
Dear chaudhary6-1,
I require a further information on how you want the login and register part. Is it possible if I use the login/register tab in the header section and redirect the user to the login/register page; please do provide me some suggestion if you have any.
Thank you and warm regards.

**Comment by chaudhary6-1** [comment]
yes you can. You could do it just like all other navigation buttons (home, about us, ...)  and you could do like only sign up (or register, no need to be showing login too at the same time).  or if it is visually better then you can also do like subscribe to our news letter (in footer) rather than sign up or login
regards
Zain Khurram Chaudhary

**Comment by shresdee** [comment]
Dear Zain Khurram Chaudhary,
I have created the subscribe in an header section like home and categories; which will redirect you to the subscribe to the news letter page with a form to fill Name, address and email and subscribe button. I am asking you whether you require other format then please let me know. Additionally the contact form will also have the similar fields. 
Thank you and warm regards

**Comment by obadaghozlan** [comment]
Hello Shresdee,
would you please send a screenshot of the webpage if possible.
best Regards,

**Comment by shresdee** [comment]
This requires a styling so please let me know what you require with the each page.
Thank you
![contact](https://user-images.githubusercontent.com/75668421/103485631-0ec5bc00-4df8-11eb-927e-8cf36b19a711.png)
![subscribe](https://user-images.githubusercontent.com/75668421/103485640-15543380-4df8-11eb-84c6-e1c3b6ee2d21.png)
